packages/titanium-address-input/src/titanium-address-input.ts

```diff
@@ -7,16 +7,58 @@ import { Address } from './Address';
 import { ManualAddressDialogElement } from './manual-address-dialog';
 import { GoogleAddressInput } from './google-address-input';
 
+/**
+ * An input with built-in google address search and manual address entering. .
+ *
+ * @element titanium-address-input
+ *
+ * @fires location-changed - Fired when address changes. details contains the selected item <T> or null.
+ */
 @customElement('titanium-address-input')
 export class TitaniumAddressInput extends LitElement {
+  /**
+   *  Displays error state if no image is empty and input is blurred.
```

Comment:
This shouldn't say image, I think it was copied from titanium-image-input, just noticed it in some other spots too